src/chrome/locale/en/https-everywhere.dtd

```diff
@@ -9,12 +9,14 @@
 <!ENTITY https-everywhere.menu.observatory "SSL Observatory Preferences">
 <!ENTITY https-everywhere.menu.globalEnable "Enable HTTPS Everywhere">
 <!ENTITY https-everywhere.menu.blockUnencryptedRequests "Block all unencrypted requests">
-<!ENTITY https-everywhere.menu.showCounter "Show Counter">
 <!ENTITY https-everywhere.menu.viewAllRules "View All Rules">
 
 <!ENTITY https-everywhere.options.generalSettings "General Settings">
+<!ENTITY https-everywhere.options.advancedSettings "Advanced Settings">
 <!ENTITY https-everywhere.options.importSettings "Import Settings">
 <!ENTITY https-everywhere.options.import "Import">
+<!ENTITY https-everywhere.options.showCounter "Show Counter">
```

Comment:
This was initially in the menu for the old XPCOM extension, which is why it is under the "menu" section.  However, changing it to reflect the latest location will clear existing translations, since it is will be tagged by another variable name in transifex.  I'd keep it how it was.